cdc/model/sink.go

```diff
@@ -332,11 +346,43 @@ type RowChangedEvent struct {
 	ReplicatingTs Ts `json:"-" msg:"-"`
 }
 
+// txnRows represents a set of events that belong to the same transaction.
+type txnRows []*RowChangedEvent
```

Comment:
How about renaming it to `rowArray` ? There is similar type declaration in the Sarama named as `recordArray` to represent all message records in the same batch.

At the moment, all rows come from the same transaction, but I thinks it's no need to mention this in the variable name.